### DISCUSSION TITLE
Transmission 4.0.3 MacOS: Torrent downloads slow down dramatically between 98-100% complete

### DISCUSSION THREAD

**Discussion by TheRoyalScam** [transmission/transmission#5451] [Q&A] [answered]
### What is the issue?

I'm sure it's probably a setting or something (I pretty much have run default for years), but I've noticed since v4.0 that torrents (magnets and actual torrent files) connect well, download relatively quickly (obvs depending on available peers), but dramatically slow down toward the completion of downloading.

Like, it'll get to around 98-99% rapidly, then take almost a full minute or more to get from 99-100%. I've never really noticed that slowdown before v4.0

Not a dealbreaker, but annoying, if it could be easily remedied.

(I'm not a coder, just a simple user...please talk "dumb" to me. ;-) )

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.3 (6b0e49bbb2)

**Comment by ckerr** [comment, marked as answer]
During steady state, there is enough missing data that there are enough block requests to make to every peer.

During endgame, there's not. So Transmission will request blocks from fewer peers. This lowers overall download speed.

**Reply by TheRoyalScam** [reply]
Thanks, Charles! Explanation makes sense. But, why haven't I noticed it before in any of the v3 app versions? I really feel a vast difference.

**Comment by fusioncy** [comment]
Is a feature to send a quick note to seed for the last 2%? thanks
